names.csv

```diff
@@ -1144,3 +1144,96 @@ zack,zach,zak
 zebedee,zeb
 zedediah,dyer,zed,diah
 zephaniah,zeph
+adam,adan 
+alberto,beto
+alejandro,alex,alexander
+alfredo,fredi,alfred
+andres,andrew
+anibal,hannibal
+antonio,tono,tony,anthony
+armando,nando,herman
+arturo, arthur
+bernardo,berna,bernard
+candelario,candel
+carlos,charles
+cristian,cris,christian
+cristobal,cris,christopher

```

Comment:
Is Christopher a nickname for Cristobal?

Reply:
"Cristóbal" is the Spanish equivalent of "Christopher." Spanish-speaking people who move to the U.S. often adopt a more English-friendly name, and this is one of those cases.